features/pom.xml

```
@@ -39,60 +39,28 @@
     </modules>
 
     <properties>
-        <!--START versions for camel features in platform-app features
-        Copied from apache-camel-2.19.4-src/parent/pom.xml
+        <!-- START versions for camel features
+        Copied from org.apache.camel:camel-parent:pom:${camel.version}
```

✏️ I prefer to use the actual version for the file from which these properties were copied from in case the camel version is upgraded, but this file isn't updated correspondingly. Like we do in the camel feature.xml file.
```suggestion
        Copied from org.apache.camel:camel-parent:pom:3.4.4
```